benchmark/parse_output.py

```diff
@@ -89,6 +89,40 @@ def parse_consel(input_f):
     return pvalues
 
 
+def parse_output(hgt_results_fp, method):
+    """Call parse_hgts() based on HGT detection method used.
+
+    Parameters
+    ----------
+    hgt_results_fp: string
+        filepath to detected HGTs
+
+    method: string
+        tool used to detect HGTs
+
+    Returns
+    -------
+    output: string
+        number of HGTs detected
+    """
+    with open(hgt_results_fp, 'r') as input_f:
+        if (method == 'ranger-dtl' or
+                method == 'trex' or
+                method == 'jane4' or
+                method == 'riata-hgt'):
+            output = parse_hgts(input_f=input_f,
+                                method=method)
+        elif method == 'consel':
+            output = parse_consel(input_f=input_f)
+            if output is None:
```

Comment:
it seems as though the return from `parse_consel` differs from `parse_hgts` which impacts its composability as it needs a special case. Would it make sense to push this `if` statement into `parse_consel` itself?